person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :id, :name, :age
+
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
```

Comment:
- [ ] Specifying `private` at the level means that the `of_age?` and `can_use_services?` methods are both `private` which is not the goal of your implementation. All methods after the `private` are executed as` private`